pipeline_steps/tempest.groovy

```diff
@@ -32,29 +32,27 @@ def tempest(){
   common.conditionalStage(
     stage_name: "Tempest Tests",
     stage: {
-      try{
-        def result = tempest_run()
-        def second_result = ""
-        if(result.contains("Race in testr accounting.")){
-          second_result = tempest_run()
-        }
-        if(second_result.contains("Race in testr accounting.")) {
-          currentBuild.result = 'FAILURE'
-        }
+      retry(2){
+        try{
+          def result = tempest_run()
+          if(result.contains("Race in testr accounting.")) {
+            currentBuild.result = 'FAILURE'
```

Comment:
Would it make more sense to put the retry with the try block so that the finally block only happens once?